### DISCUSSION TITLE
How does differ from MobX?

### DISCUSSION THREAD

**Discussion by bichotll** [preactjs/signals#295] [Q&A]
It seems like MobX is quite similar and has some extra enhancements?

Don't take me wrong, it's nice to see another similar library that makes things easy-peasy.

**Comment by dcporter** [comment]
(My very small answer is that you can't just get the core out of mobx and build on it. Preact signals core is 4 kb ❤️❤️)

**Comment by rschristian** [comment]
To avoid rephrasing Jason I'll just provide his response to this very question: https://twitter.com/_developit/status/1567344948747419648
